testing/regress/ecl/h3.ecl

```diff
@@ -0,0 +1,51 @@
+IMPORT lib_h3;
```

Comment:
Should maybe have a copyright message, and a //3rdparty tag?

Reply:
Inside the regress suite or inside lib_h3.eclib?
FWIW the h3 sources are also under Apache 2.0 license which is included in the h3 submodule AND non of our other plugins appear to reference there dependent licenses?

Reply:
I was meaning that maybe the regression test file should have a copyright header (some but not all others do), and that a comment in the regression test file of the form

//class=3rdparty

might be appropriate (they are used when deciding what subsets of the regression suite to run).